sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clientTelemetry/ClientTelemetry.java

```diff
@@ -94,15 +94,15 @@ public ClientTelemetryInfo getClientTelemetryInfo() {
         return clientTelemetryInfo;
     }
 
-    public static void recordValue(DoubleHistogram doubleHistogram, long value) {
+    public static void recordValue(ConcurrentDoubleHistogram doubleHistogram, long value) {
```

Comment:
potential perf implication.

Reply:
No perf implication , this is just reference which i change from DoubleHistogram to ConcurrentDoubleHistogram , object was already ConcurrentDoubleHistogram in the code.